server/project-service/src/main/java/edu/illinois/ncsa/incore/service/project/controllers/ProjectController.java

```diff
@@ -848,9 +892,10 @@ public List<WorkflowResource> listWorkflowsOfProject(
     }
 
     @GET
+    @Deprecated
     @Path("{projectId}/workflows/search")
     @Produces(MediaType.APPLICATION_JSON)
-    @Operation(description = "Search workflows from a project")
+    @Operation(description = "Search workflows from a project", deprecated = true)
```

Comment:
Add the message for deprecation?

Reply:
The deprecation actually doesn't work. 
I think I will just remove all of those endpoints to avoid confusion.